services/auth/source/ldap/source_group_sync.go

```diff
@@ -20,22 +22,15 @@ func (source *Source) SyncLdapGroupsToTeams(user *user_model.User, ldapTeamAdd,
 	for orgName, teamNames := range ldapTeamAdd {
 		org, ok := orgCache[orgName]
 		if !ok {
-			org, err = models.GetOrgByName(orgName)
+			org, err = organization.GetOrgByName(orgName)
 			if err != nil {
 				// organization must be created before LDAP group sync
 				log.Warn("LDAP group sync: Could not find organisation %s: %v", orgName, err)
 				continue
 			}
 			orgCache[orgName] = org
 		}
-		if isMember, err := models.IsOrganizationMember(org.ID, user.ID); !isMember && err == nil {
-			log.Trace("LDAP group sync: adding user [%s] to organization [%s]", user.Name, org.Name)
-			err = org.AddMember(user.ID)
-			if err != nil {
-				log.Error("LDAP group sync: Could not add user to organization: %v", err)
-				continue
-			}
-		}
```

Comment:
Code removed by purpose?

Reply:
Yes, `org.AddMember` has gone. When add a user as a team, it will automatically add user as org member, when deleting a user from the last team on the org, he will be removed from that org automatically. So we don't need to a standalone method to add user to organization.